### PR TITLE
Bugfix: registerBlockType filter

### PR DESCRIPTION
**Pull request by alexander-kozhukhovski**
## What does this do/fix?

- Provides possible solution for `blocks.registerBlockType` filter [issue](https://github.com/moderntribe/square-one/issues/1026)
- Provides an [example](https://github.com/moderntribe/square-one/pull/1036/files#r949348577) of removing align settings for `core/separator` block
- Provides an [example](https://github.com/moderntribe/square-one/pull/1036/files#r949348577) of filtering align settings for `core/embed` block

## QA

Links to relevant issues
- [Link to Issue](https://github.com/moderntribe/square-one/issues/1026)

## Tests

Does this have tests?

- [ ] Yes
- [x] No, this doesn't need tests
- [ ] No, I need help figuring out how to write the tests.

### CODE REVIEW

File: wp-content/themes/core/assets/js/src/admin/editor/register-block-filter.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { addFilter } from '@wordpress/hooks';
+
+// Specify block name and align settings for it to keep.
+const blockAlignments = {
```
Comment (alexander-kozhukhovski) [inline wp-content/themes/core/assets/js/src/admin/editor/register-block-filter.js:4]: Example

**Review by Vinsanity — APPROVED** [review]
Nice work Alex. This will be super helpful when adjusting block settings. This is also a safe bet right now on the embed block due to the issues with Gutenberg and the discrepancies between admin and theme markup.

**Review by ChrisMKindred — APPROVED** [review]
👍 Thanks!

**Comment by ChrisMKindred** [comment]
@alexander-kozhukhovski Can you add a changelog entry, please?

**Comment by alexander-kozhukhovski** [comment]
> @alexander-kozhukhovski Can you add a changelog entry, please?

Changelog entry added

**Pull request merged**